src/main/java/models/Game.java

```diff
@@ -1,51 +1,211 @@
 package models;
 
+import java.util.Collections;
+import java.util.ArrayList;		//game is an array of arrays
+import java.util.Random;		
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Random;
 
-/**
-* Created by IStallcup on 2/27/16.
-*/
 
 
-//abstract class
-public class Game {
+//game class - has a card
+//player has a game
+
+
+
+public class Game implements Serializable{
+
+
+	//fields
+
+	public java.util.List<Card> deck;		//initialize an array of cards called Deck
+
+	public Player a_player;		
+	public Dealer a_dealer;
+	c boolean errorFlag = false;
+	public String userMessage = "Place your Bet";
+	public String userMessageTwo = "";
 
-	Random rnd = new Random();
+	public boolean againDisabled = true;
+	public boolean dealDisabled = false;
+	public boolean hitDisabled = true;
+	public boolean standDisabled = true;
+	public boolean splitDisabled = true;
+	public boolean doubleDisabled = true;
+	public boolean bettingDisabled = false;
+	public boolean previousTie = false;
+	public boolean doubleDownTwoDisabled = false;
+	public boolean hasSplit = false;
+	public boolean hitTwoDisabled = false;
+	public boolean standTwoDisabled = false;
+	public boolean secondStand = false;
+	public boolean firstStand = false;
+
+
+
+
+	//methods
+	public Game(){
+		this.player = new player(1);
+		this.dealer = new dealer();
+		this.deck = new ArrayList<>();
+		buildDeck();
+
+
+	}
 
-	//create the deck list
-	public java.util.List<Card> deck = new ArrayList<>();
 
-	//use int as placeholder as the suit of the card doesn't matter in blackjack
 
-	public void buildDeck()
+	//build a deck of cards
+	public void build()
```

Comment:
Why do you have a build deck function here if there's already another in use below?